blog-posts/Learning-about-git/Learning-about-git.md

```diff
@@ -0,0 +1,112 @@
+---
+published: false
+title: 'Learning about git!'
+cover_image: 'https://raw.githubusercontent.com/ramclen/documenting-dev/master/blog-posts/Learning-about-git/assets/cover-image.jpg'
+description: 'Learning basics about git'
+tags: git, beginners, productivity
+series:
+canonical_url:
+---
+
+In this post, I will try to give you some concepts to help you to start learning about git. Git is known by almost all programmers or even all of them. It manages all the changes made to your files and projects in an easy way.
+
+A way I love to see git is like a save state on a video game, if I fail in next boss I will restart from my previous state and I will give another try with a different strategy.
```

Comment:
"A way" doesn't sound good to me. Maybe is better: I love to see git like a save state on video game...